util/syspolicy/handler_windows.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package syspolicy
+
+import "tailscale.com/util/winutil"
+
+type windowsHandler struct{}
+
+func init() {
+	RegisterHandler(windowsHandler{})
+}
+
+func (windowsHandler) ReadString(key string) (string, error) {
+	return winutil.GetPolicyString(key)
```

Comment:
Where do we convert `winutil.ErrNoValue` to `ErrNoSuchKey`?

Reply:
Oh, good point. Previously this file had `var ErrNoSuchKey = winutil.ErrNoValue` and it was defined to something else for other OSes, but that's gone now.